tests/manage/mcg/test_bucket_deletion.py

```diff
@@ -1,50 +1,214 @@
 import logging
 
+import timeit
+import botocore
 import pytest
 
 from ocs_ci.framework.pytest_customization.marks import (
-    tier1, noobaa_cli_required, acceptance,
+    tier1, tier2, tier3, noobaa_cli_required, acceptance,
     filter_insecure_request_warning
 )
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.mcg_bucket import S3Bucket, OCBucket, CLIBucket
+from ocs_ci.utility.utils import run_mcg_cmd
+from tests.helpers import create_unique_resource_name
+from tests.manage.mcg import helpers
 
 logger = logging.getLogger(__name__)
 
 
 @filter_insecure_request_warning
-@acceptance
-@tier1
 class TestBucketDeletion:
     """
     Test bucket Creation Deletion of buckets
     """
-    @pytest.mark.polarion_id("OCS-1299")
-    def test_s3_bucket_delete(self, mcg_obj, bucket_factory):
+    @pytest.mark.parametrize(
+        argnames="amount,interface",
+        argvalues=[
+            pytest.param(
+                *[3, 'S3'],
+                marks=[pytest.mark.polarion_id("OCS-1939"), tier1, acceptance]
+            ),
+            pytest.param(
+                *[3, 'CLI'],
+                marks=[tier1, acceptance, noobaa_cli_required,
+                       pytest.mark.polarion_id("OCS-1940")]
+            ),
+            pytest.param(
+                *[3, 'OC'],
+                marks=[tier1, acceptance, pytest.mark.polarion_id("OCS-1299")]
+            ),
+            pytest.param(
+                *[100, 'S3'],
+                marks=[tier2, pytest.mark.polarion_id("OCS-1865")]
+            ),
+            pytest.param(
+                *[100, 'OC'],
+                marks=[tier2, pytest.mark.polarion_id("OCS-1915")]
+            ),
+            pytest.param(
+                *[1000, 'S3'],
```

Comment:
I think it'd be best to update it to conform with #1176 by skipping the test (skipif True) and stating the erratic behavior issue

Reply:
Updated accordingly